stock_picking_invoice_link/models/stock_picking.py

```diff
@@ -17,9 +17,35 @@ class StockPicking(models.Model):
     invoice_id = fields.Many2one(
         comodel_name='account.invoice', string='Invoice',
         compute="_compute_invoice_id")
+    invoice_exists = fields.Boolean(
```

Comment:
The only drawback I can see is showing a button on the form while there is no related invoice if this module has been installed on an existing database. Do you prefer removing this field anyway?